docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-2.1-applications-linux-containers-aks-kubernetes.md

```diff
@@ -0,0 +1,180 @@
+
+```console
+docker tag mssampleaksapplication:latest mssampleacr.azurecr.io/mssampleaksapplication:v1
+```
+
+After running the `docker tag` command, list the images with the `docker images` command, you should see the image with the new tag.
+
+![Console output from the docker images command.](media/tagged-docker-images-list.png)
+
+**Figure 4-44**. View of tagged images
+
+### Push the image into the Azure ACR
+
+Now you can push the image into the Azure ACR, using the command:
+
+```console
+docker push mssampleacr.azurecr.io/mssampleaksapplication:v1
+```
+
+This command takes a while uploading the images but gives you feedback in the process.
+
+![Console output from the docker push command: shows a character based progress bar for each layer.](media/uploading-image-to-acr.png)
+
+**Figure 4-45**. Uploading the image to the ACR
+
+You can see below the result you should get when the process completes:
+
+![Console output from the docker push command, finished, showing all layers or nodes.](media/uploading-docker-images-complete.png)
+
+**Figure 4-46**. View of nodes
+
+The next step is to deploy your container into your AKS Kubernetes cluster, for that you need a file (**.yml deploy file**) that, in this case, contains:
+
+![Kubernetes deploy file content.](media/kubernetes-deploy-file.png)
+
+> [!More info]
+> For more information on this topic, see: [https://kubernetes.io/docs/user-guide/kubectl-cheatsheet/](https://kubernetes.io/docs/user-guide/kubectl-cheatsheet/)
+
+Now you're almost ready to deploy using **Kubectl**, but first you must get the credentials to the AKS Cluster with this command:
+
+```console
+az aks get-credentials --resource-group MSSampleResourceGroupAKS --name mssampleclusterk801
+```
+
+![Console output from the above command: Merged "MSSampleK8Cluster as current context in  /root/.kube/config](media/getting-aks-credentials.png)
+
+**Figure 4-47**. getting credentials
+
+Then, use the `kubectl create` command to launch the deployment.
+
+```console
+kubectl create -f mssample-deploy.yml
+```
+
+![Console output from the above command: deployment "mssamplesbook" created. service "mssample-kub-app" created.](media/kubectl-create-command.png)
+
+**Figure 4-48**. Deploy to Kubernetes
+
+When the deployment completes, you can access the Kubernetes console with a local proxy that you can temporally access with this command:
+
+```console
+az aks browse --resource-group MSSampleResourceGroupAKS --name mssampleclusterk801
+```
+
+And accessing the url [http://127.0.0.1:8001](http://127.0.0.1:8001).
+
+![Browser view of the Kubernetes dashboard, showing Deployments, Pods, Replica Sets, and Services.](media/kubernetes-cluster-information.png)
+
+**Figure 4-49**. View Kubernetes cluster information
+
+Now you have your application deployed on Azure, using a Linux Container, and an AKS Kubernetes Cluster that you can access browsing to the public IP of your service that you can get from the Azure portal.
```

Comment:
Cluster -> cluster

It would be good to break this sentence.

 to the public IP of your service that you can get ->  to the public IP of your service, which you can get

Reply:
fixed